ipt_connect/IPTdev/templates/IPTdev/ranking.html

```diff
@@ -2,22 +2,31 @@
 {% load humanize %}
 {% load i18n %}
 {% block content %}
+  
+  {% if not rankteams %}
+      <div>
+        <center>
+        <h1>{% trans 'No teams are registered yet' %}</h1>
```

Comment:
English typo: 'No team registered yet' or 'No team has been registered yet'
Same in the following commits.

Everything else seems fine!